### DISCUSSION TITLE
cannot found cmd command pyinstaller  in windows

### DISCUSSION THREAD

**Discussion by data2** [pyinstaller/pyinstaller#8789] [PyInstaller] [answered]
after pip install pyinstaller, i cannot found command pyinstaller in cmd windos ? what should i do ?

![image](https://github.com/user-attachments/assets/e7a59534-6704-4308-aa37-a41807cad951)

![image](https://github.com/user-attachments/assets/118bc515-d26c-4ae8-abe3-cba345cb57b8)

**Comment by data2** [comment, marked as answer]
anybody help me , thx

**Reply by rokm** [reply]
https://pyinstaller.org/en/stable/installation.html#troubleshooting-missing-pyinstaller-command
